package.json

```diff
@@ -92,8 +92,8 @@
     "isomorphic-webcrypto": "^2.3.4",
     "loadjs": "^3.6.1",
     "lodash": "^4.17.15",
-    "lottie-ios": "^3.1.6",
-    "lottie-react-native": "^3.3.2",
+    "lottie-ios": "^2.5.3",
+    "lottie-react-native": "^2.6.1",
```

Comment:
based on what?
here they say they have it working
https://github.com/react-native-community/lottie-react-native/issues/581